lib/OpModel/TTNN/TTNNOpModelLib_disabled.cpp

```diff
@@ -0,0 +1,25 @@
+// SPDX-FileCopyrightText: (c) 2024 Tenstorrent AI ULC
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "TTNNOpModel.h"
+
+namespace mlir::tt::op_model::ttnn {
+
+//===----------------------------------------------------------------------===//
+// ReluOp
+//===----------------------------------------------------------------------===//
+
+bool ReluOpInterface::isLegal(
+    const mlir::tt::ttnn::TTNNLayoutAttr &inputLayout,
+    const mlir::tt::ttnn::TTNNLayoutAttr &outputLayout) {
+  return true;
+}
```

Comment:
Why need for redefinition in another file, because of mac build? Can we use ifdef macro instead?

Reply:
apple build and if feature is disabled. other file was cleaner in my mind. I can merge if that's your preference

Reply:
Yeah please do so whole logic and switch can be observed in single place.

Reply:
done